EPRN.Portal/Views/Prns/Action.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@using EPRN.Common.Enums;
+@using EPRN.Portal.Services.Interfaces;
+@using EPRN.Portal.Helpers.Extensions;
+
+@inject IUserRoleService userRoleService;
+
+@model ActionPrnViewModel
+@{
+    ViewData["Title"] = ActionPRNResources.Title;
+}
+
+<partial name="_ValidationSummary" />
+<form method="post">
+    <input type="hidden" asp-for="JourneyId" />
```

Comment:
This is not needed. The Id field will be populated from the route during binding

Reply:
Done.